R/cfr_time_varying.R

```diff
@@ -48,11 +48,11 @@
 #' for each \eqn{t}, where \eqn{\theta} represents the severity measure of
 #' interest.
 #'
-#' The epidemiological delay distribution passed to `epidist` is used to obtain
-#' a probability mass function parameterised by time; i.e. \eqn{f(t)} which
-#' gives the probability a case has a known outcomes (usually, death) at time
-#' \eqn{t}, parameterised with disease-specific parameters before it is supplied
-#' here.
+#' The epidemiological delay distribution passed to `delay_density` is used to
+#' obtain a probability mass function parameterised by time; i.e. \eqn{f(t)}
+#' which gives the probability a case has a known outcomes (usually, death) at
+#' time \eqn{t}, parameterised with disease-specific parameters before it is
```

Comment:
```suggestion
#' which gives the probability of the binary outcome of a case (usually, survival or death) being known by
#' time \eqn{t}. The delay distribution is parameterised with disease-specific parameters before it is
```

Reply:
This is an improvement in clarity, however, if the outcome of a case is binary can the "usually" be removed, doesn't it always have to be survival or death?

Reply:
Agreed - happy for it to be removed. Although to be very nitpicky there's a question about whether the distribution is of onset-to-death-occurence or onset-to-death-reporting. I guess that as long as the distributions used are consistently one or the other it doesn't matter too much, if that makes sense.